applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp

```diff
@@ -134,6 +134,8 @@ void PrestressMembraneElement::Initialize()
     mThickness0 = GetProperties()[THICKNESS];
```

Comment:
since THICKNESS is in the properties, there is A SINGLE ONE for all the elements, hence it is incorrect to change it